src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -47,6 +47,11 @@
     public static final String PLUGIN_NAME = "knn";
     public static final String MODEL_METADATA_FIELD = "knn-models";
 
```

Comment:
FAR: we can sort constants in ascending order

Reply:
Agree #96